glue/plugins/scatter_group/qt_widget.py

```diff
@@ -0,0 +1,154 @@
+from __future__ import print_function, division
+import os.path
+
+from ...qt.widgets.scatter_widget import ScatterWidget
+from ...qt.widgets.mpl_widget import defer_draw
+from ...qt.widget_properties import (ButtonProperty, FloatLineProperty,
+                                 CurrentComboProperty,
+                                 connect_bool_button, connect_float_edit)
+
+from ...qt.qtutil import nonpartial, load_ui
+from .client import ScatterGroupClient
+
+__all__ = ['ScatterGroupWidget']
+
+
+class ScatterGroupWidget(ScatterWidget):
+
+    LABEL = "Scatter Group Plot"
+    _property_set = ScatterWidget._property_set + ['gatt']
+
+    xlog = ButtonProperty('ui.xLogCheckBox', 'log scaling on x axis?')
+    ylog = ButtonProperty('ui.yLogCheckBox', 'log scaling on y axis?')
+    xflip = ButtonProperty('ui.xFlipCheckBox', 'invert the x axis?')
+    yflip = ButtonProperty('ui.yFlipCheckBox', 'invert the y axis?')
+    xmin = FloatLineProperty('ui.xmin', 'Lower x limit of plot')
+    xmax = FloatLineProperty('ui.xmax', 'Upper x limit of plot')
+    ymin = FloatLineProperty('ui.ymin', 'Lower y limit of plot')
+    ymax = FloatLineProperty('ui.ymax', 'Upper y limit of plot')
+    hidden = ButtonProperty('ui.hidden_attributes', 'Show hidden attributes')
+    xatt = CurrentComboProperty('ui.xAxisComboBox',
+                                'Attribute to plot on x axis')
+    yatt = CurrentComboProperty('ui.yAxisComboBox',
+                                'Attribute to plot on y axis')
+    gatt = CurrentComboProperty('ui.groupComboBox',
+                                'Attribute to group time series')
+
+    def _load_ui(self):
+        self.ui = load_ui(os.path.join(os.path.dirname(__file__), 'scattergroupwidget.ui'), self.option_widget)
```

Comment:
@ChrisBeaumont - right, I need to check that. I had solved that issue as part of https://github.com/glue-viz/glue/pull/552 so I will take a look this evening to see what I had done there.